src/components/SelectionTable/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import rows from './data';
+import columns from './columns';
+import {DataGrid, DataGridProps} from "@mui/x-data-grid";
+
+export interface historyElement {
+    companyId?: number;
+    dataAsAt?: string;
+    firstYear?: number;
+    id?: number;

```

Comment:
make id, firstYear, lastYear as required parameters. Do we really need all other unused params?